pkg/cover/instrument.go

```diff
@@ -131,11 +131,15 @@ func clearFileCover(counter []uint32) {
 }
 
 func registerHandlers() {
+	{{if .Singleton}}
+	ln, _, err := listen()
```

Comment:
The purpose of this statement seems to avoid the scenario that in Singleton mode, the host variable is defined, but not used, right?

if yes, i prefer to print extra information to enhance it, like:

``` go
        ln, host, err := listen()
	if err != nil {
		log.Fatalf("listen failed, err:%v", err)
	}
        profileAddr := "http://" + host
        log.Debugf("[goc] goc profile address: %s \n", profileAddr)

Reply:
good practice, i'll make change

Reply:
seems this comment was not resolved, what's happening?

Reply:
log output will impact e2e test that assert gocc output